components/layout/stylesheets/details.css

```
@@ -0,0 +1,18 @@
+/* We should use `content-visibility` but it is yet to be implemented.
+ * These rule would not comply with ::details-content and should be removed
+ * with it's implementation */
+slot[id="internal-contents-slot"] {
+  display: none;
+}
+:host([open]) slot[id="internal-contents-slot"] {
+  display: block;
+}
+
+summary[id="internal-fallback-summary"] {
```

```suggestion
#internal-fallback-summary::slotted(summary) {
```
Because we don't just style the fallback summary. Ditto below.

---

I am unsure whether it could match. Probably it was matching the element in the light tree. 

I have changed it to be `:host summary:first-of-type` and `:host([open]) summary:first-of-type`.  Since `:host` only matches node in shadow DOM, I suppose selecting it with an identifier is not necessary.